app/src/main/java/org/schabi/newpipe/fragments/MainFragment.java

```diff
@@ -126,7 +170,89 @@ public CharSequence getPageTitle(int position) {
 
         @Override
         public int getCount() {
-            return this.tabTitles.length;
+            if(PreferenceManager.getDefaultSharedPreferences(getActivity())
+                    .getString(getString(R.string.main_page_content_key), getString(R.string.blank_page_key))
+                    .equals(getString(R.string.subscription_page_key))) {
+                return 1;
+            } else {
+                return 2;
+            }
+        }
+    }
+
+    /*//////////////////////////////////////////////////////////////////////////
+    // Main page content
+    //////////////////////////////////////////////////////////////////////////*/
+
+    private Fragment getMainPageFramgent() {
+        try {
+            SharedPreferences preferences =
+                    PreferenceManager.getDefaultSharedPreferences(getActivity());
+            final String set_main_page = preferences.getString(getString(R.string.main_page_content_key),
```

Comment:
`set_main_page` should be in camel case.

Reply:
fixed